crest/Assets/Crest/Crest/Shaders/OceanEmission.hlsl

```diff
@@ -70,7 +70,8 @@ void ApplyCaustics
 {
 	// could sample from the screen space shadow texture to attenuate this..
 	// underwater caustics - dedicated to P
-	const float3 scenePosUV = WorldToUV(i_scenePos.xz, cascadeData1, _LD_SliceIndex + 1);
+	// TODO: Investigate if necessary anymore.
+	const float3 scenePosUV = WorldToUV(CREST_WITH_FLOATING_ORIGIN_LOD_OFFSET_XZ(i_scenePos.xz), cascadeData1, _LD_SliceIndex + 1);
```

Comment:
this is the only case where this FO macros is used going into WorldToUv, it might be good to document why its needed here. or maybe thats the TODO?

Reply:
No longer needed. Fixed in C#. Thanks :^)